velox/functions/prestosql/window/tests/NthValueTest.cpp

```diff
@@ -124,46 +148,6 @@ TEST_F(NthValueTest, multiInputWithSortOrder) {
       doubleVectors, "nth_value(c0, 25)", kSortOrderBasedOverClauses);
 }
 
-TEST_F(NthValueTest, integerValues) {
```

Comment:
Because these tests were not parameterized on frameClause earlier, they were moved later in the file. They are now parameterized and in the original position.